packages/meteor-gazelle-ip-manager/lib/methods.js

```diff
@@ -0,0 +1,123 @@
+IpUtils = {
+  VALID_IP_FORMAT: /^([1-9]\d*|0[0-7]*|0x[\da-f]+)(?:\.([1-9]\d*|0[0-7]*|0x[\da-f]+))?(?:\.([1-9]\d*|0[0-7]*|0x[\da-f]+))?(?:\.([1-9]\d*|0[0-7]*|0x[\da-f]+))?$/i,
+  ip2long: function (ipAddr) {
+    // TODO(rhomes) needs to handle IPv6 addresses
+    var i = 0;
+    ipAddr = ipAddr.match(this.VALID_IP_FORMAT);
+    if (!ipAddr) {
+      return false;
+    }
+
+    ipAddr[0] = 0;
```

Comment:
Can you explain the whole ip2long function. 
Or leave the comments from http://phpjs.org/functions/ip2long/
They are really useful to understand.